CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+## 2.13.0-rc.0 / 2019-09-25
+
+* [CHANGE] Metrics: Change prometheus_sd_configs_failed_total to Gauge #5254
+* [CHANGE] Service discovery: add new node address types for kubernetes. #5902
+* [CHANGE] Metrics format: update parsing to disallow new Go 1.13 float formats. #5984
+* [ENHANCEMENT] Remote read: use remote read streaming to reduce memory usage. #5703
+* [ENHANCEMENT] Docs: fix spelling mistakes and other small changes . #5952, #6051
+* [ENHANCEMENT] GUI: show warnings if query have returned some warnings. #5964
+* [ENHANCEMENT] Reduce memory usage: cache labels.Labels to Identify Series in Remote Write. #5849
```

Comment:
(suggestion) `Remove write: reduce memory usage by caching labels to identify series.`

Reply:
good suggestion, thanks

Reply:
We were already caching the labels, this PR just changed the internal format. Perhaps:
`Remote write: reduce memory usage of the series cache.`

Reply:
updated